data/seed/Seeding.md

```diff
@@ -0,0 +1,99 @@
+# Seeding the Database
+
+This covers seeding the database for development work.
+
+## Preparation
+
+You should have followed the steps for getting started and have:
+
+- PostgreSQL database and Elasticsearch search engine running in Docker containers.
+- Vite front end server running in another terminal window.
+- Django development server running in another terminal window.
+
+At this point, you should be able to login to the Admin website at
+http://localhost:8000/ or http:127.0.0.1:8000/ with the email address and
+password of the super user you created.
+
+In a fourth terminal window, have your Python virtual environment activated and
+be in your working directory with the UIPA.org source files cloned from your
+fork of the main branch of the [UIPA.org
+repository](https://github.com/CodeWithAloha/uipa).
+
+## Seeding
+
+To seed the database, run these commands:
+
+```
+$ python manage.py loaddata data/seed/new-classification.json                                
```

Comment:
Can we timestamp these files? Something like `2024-03-21-classification.json`

The format we had previously: https://github.com/CodeWithAloha/uipa/tree/master/data

Reply:
Yes, will do.